lib/asn1/src/asn1ct_gen.erl

```diff
@@ -1327,9 +1327,25 @@ gen_head(#gen{options=Options}=Gen, Mod, Hrl) ->
 	0 -> ok;
 	_ -> emit(["-include(\"",Mod,".hrl\").",nl])
     end,
+    Deterministic = proplists:get_bool(deterministic, Options),
+    Options1 =
+        case Deterministic of
+            true ->
+                % compile:keep_compile_option will filter some of these
+                % out of generated .beam files, but this will keep
+                % them out of the generated .erl files
```

Comment:
```suggestion
                %% compile:keep_compile_option will filter some of these
                %% out of generated .beam files, but this will keep
                %% them out of the generated .erl files
```